File: Ex25ListViewCustom/app/src/main/java/com/gy25m/ex25listviewcustom/Item.java
package com.gy25m.ex25listviewcustom;

public class Item {
    String name;     //전현무
    String nation;  //대한민국
    int imgId;     //R.drawble.korea 태극기

    //객체를 생성할때 자동으로 실행되는 특별한 메소드-Constructor
    public Item(String name,String nation,int imgId){
        this.name=name;
        this.nation=nation;
        this.imgId=imgId;
    }
}

File: Ex92KotlinRecyclerView/app/src/main/java/com/gy25m/ex92kotlinrecyclerview/Item.kt
package com.gy25m.ex92kotlinrecyclerview

// data class: 데이터만 저장하는 목적의 클래스 - 일반 클래스와 다르게
// 자동으로 equals()할때 객체 주소를 비교하지 않고 멤버값들을 비교해주도록 오버라이드 된 클래스
// 주생성자에 있는거만 equals함
data class Item constructor (var name:String,var msg:String,var imgId:Int ) {   //constructor라는 말은 생략가능 ()이거말고

}